### ISSUE TITLE
Remove output to stdout/stderr for command in network-manager

### ISSUE THREAD

**Issue by leodotcloud** [rancher/rancher#7611]
This gets printed multiple times `net.ipv4.conf.docker0.route_localnet = 1`. Clean this up.

**Comment by sangeethah** [comment]
Info level logs appears as part of  "Standard Error"
```
20/2017 4:57:12 PMtime="2017-03-20T23:57:12Z" level=info msg="CNI up done" cid=de38e8fe6670994d647cef7eaf7fc039cc6bfb2e403fd844e446b91cdcd7e841 networkMode=ipsec result=IP4:{IP:{IP:10.42.178.33 Mask:ffff0000} Gateway:10.42.0.1 Routes:[{Dst:{IP:169.254.169.250 Mask:ffffffff} GW:<nil>} {Dst:{IP:0.0.0.0 Mask:00000000} GW:10.42.0.1}]}, DNS:{Nameservers:[] Domain: Search:[] Options:[]}
3/20/2017 4:57:12 PMtime="2017-03-20T23:57:12Z" level=info msg="CNI up done" cid=8a9ee18bbd412c7a164af9a4c6e2385a6589f824342b543d83b0c9fa84e32722 networkMode=ipsec result=IP4:{IP:{IP:10.42.118.125 Mask:ffff0000} Gateway:10.42.0.1 Routes:[{Dst:{IP:169.254.169.250 Mask:ffffffff} GW:<nil>} {Dst:{IP:0.0.0.0 Mask:00000000} GW:10.42.0.1}]}, DNS:{Nameservers:[] Domain: Search:[] Options:[]}
3/20/2017 4:57:25 PMtime="2017-03-20T23:57:25Z" level=info msg="CNI down" cid=8a9ee18bbd412c7a164af9a4c6e2385a6589f824342b543d83b0c9fa84e32722 networkMode=ipsec
3/20/2017 4:57:25 PMtime="2017-03-20T23:57:25Z" level=info msg="CNI down" cid=5f70af0585063a78b4b3fffe7e07a70e4189465b4f71030eda88d563f1a7930a networkMode=ipsec
3/20/2017 4:57:26 PMtime="2017-03-20T23:57:26Z" level=info msg="CNI down" cid=50028ba729ebb9199aad6dc167b1435b7991817a0d47e22050702ad46825bd0b networkMode=ipsec
3/20/2017 4:57:26 PMtime="2017-03-20T23:57:26Z" level=info msg="CNI down" cid=de38e8fe6670994d647cef7eaf7fc039cc6bfb2e403fd844e446b91cdcd7e841 networkMode=ipsec
3/20/2017 4:57:42 PMtime="2017-03-20T23:57:42Z" level=info msg="Setting up resolv.conf for ContainerId [97375f9cbc4033235fe2e54bea1d864a80403e8ea816c55c19df76399771ba3e]"
3/20/2017 4:57:42 PMtime="2017-03-20T23:57:42Z" level=info msg="CNI up" cid=97375f9cbc4033235fe2e54bea1d864a80403e8ea816c55c19df76399771ba3e networkMode=ipsec
3/20/2017 4:57:42 PMtime="2017-03-20T23:57:42Z" level=info msg="CNI up done" cid=97375f9cbc4033235fe2e54bea1d864a80403e8ea816c55c19df76399771ba3e networkMode=ipsec result=IP4:{IP:{IP:10.42.26.108 Mask:ffff0000} Gateway:10.42.0.1 Routes:[{Dst:{IP:169.254.169.250 Mask:ffffffff} GW:<nil>} {Dst:{IP:0.0.0.0 Mask:00000000} GW:10.42.0.1}]}, DNS:{Nameservers:[] Domain: Search:[] Options:[]}
3/20/2017 4:57:42 PMtime="2017-03-20T23:57:42Z" level=info msg="Setting up resolv.conf for ContainerId [cd76910a3fcb3209aa51e50c43271514c0dfa23992252c3dce6863dbab4c1634]"
3/20/2017 4:57:42 PMtime="2017-03-20T23:57:42Z" level=info msg="CNI up" cid=cd76910a3fcb3209aa51e50c43271514c0dfa23992252c3dce6863dbab4c1634 networkMode=ipsec
3/20/2017 4:57:42 PMtime="2017-03-20T23:57:42Z" level=info msg="CNI up done" cid=cd76910a3fcb3209aa51e50c43271514c0dfa23992252c3dce6863dbab4c1634 networkMode=ipsec result=IP4:{IP:{IP:10.42.182.14 Mask:ffff0000} Gateway:10.42.0.1 Routes:[{Dst:{IP:169.254.169.250 Mask:ffffffff} GW:<nil>} {Dst:{IP:0.0.0.0 Mask:00000000} GW:10.42.0.1}]}, DNS:{Nameservers:[] Domain: Search:[] Options:[]}
3/20/2017 4:59:22 PMtime="2017-03-20T23:59:22Z" level=info msg="Setting up resolv.conf for ContainerId [f4b0043787535c7053e004087332230863d5ca074baae5260231a40e11a3c075]"
3/20/2017 4:59:22 PMtime="2017-03-20T23:59:22Z" level=info msg="CNI up" cid=f4b0043787535c7053e004087332230863d5ca074baae5260231a40e11a3c075 networkMode=ipsec
3/20/2017 4:59:22 PMtime="2017-03-20T23:59:22Z" level=info msg="CNI up done" cid=f4b0043787535c7053e004087332230863d5ca074baae5260231a40e11a3c075 networkMode=ipsec result=IP4:{IP:{IP:10.42.73.35 Mask:ffff0000} Gateway:10.42.0.1 Routes:[{Dst:{IP:169.254.169.250 Mask:ffffffff} GW:<nil>} {Dst:{IP:0.0.0.0 Mask:00000000} GW:10.42.0.1}]}, DNS:{Nameservers:[] Domain: Search:[] Options:[]}
3/20/2017 4:59:23 PMtime="2017-03-20T23:59:23Z" level=info msg="Applying new port rules
```

**Comment by leodotcloud** [comment]
Fixed in `network-services: v0.2.6` with `rancher/network-manager:v0.7.8`.

**Comment by sangeethah** [comment]
Tested with network-services: v0.26 with rancher/network-manager:v0.7.8.
``` net.ipv4.conf.docker0.route_localnet = 1``` does not get printed in the logs repeatedly.
